src/sonic-host-services/scripts/hostcfgd

```diff
@@ -966,6 +970,67 @@ class NtpCfg(object):
             syslog.syslog(syslog.LOG_INFO, 'ntp server update, restarting ntp-config, ntp servers configured {}'.format(self.ntp_servers))
             run_cmd(cmd)
 
+class PamLimitsCfg(object):
+    """
+    PamLimit Config Daemon
+    1) The pam_limits PAM module sets limits on the system resources that can be obtained in a user-session.
+    2) Purpose of this daemon is to render pam_limits config file.
+    """
+    def __init__(self, CfgDb):
```

Comment:
>CfgDb [](http://example.com/codeflow?start=23&length=5)

We prefer variable_name_with_underscore. I know there are mixed styles in this file already. You may change only your new code.

Reply:
Fixed.